src/main/java/org/graylog/plugins/threatintel/functions/otx/AbstractOTXLookupFunction.java

```diff
@@ -39,9 +40,10 @@ private OTXLookupResult lookupIntel(final String key, final LookupTableService.F
 
             if (pulseCount > 0) {
                 result.put("otx_threat_indicated", true);
-                if (lookupResult.multiValue() != null && lookupResult.multiValue() instanceof List) {
+                if (lookupResult.multiValue() != null && lookupResult.multiValue() instanceof LinkedHashMap) {
```

Comment:
Why did you decide to use a `LinkedHashMap` instead of the `List` interface?

Reply:
The `lookupResult.multivaule()` is a instance of `LinkedHashMap` not `List` which was causing this check `if statement` to break.

Reply:
Ah, it's a map and not a list! 🤦‍♂️ 

In this case, please use the `Map` interface and not `List` or `LinkedHashMap`.

Reply:
Wiil do!